pep-8015.rst

```diff
@@ -306,6 +309,37 @@ We work toward this common goal in three ways:
 The organization of this workgroup is defined by the
 `ConductWG Charter <https://wiki.python.org/psf/ConductWG/Charter>`_.
 
+Special Case: Ban a core developer
+----------------------------------
+
+As any other member of the Python community, the PSF Code of Conduct
+Workgroup can ban a core developer for a limited amount of time. In this
+case, the core developer immediately looses its core developer status to
+become again a Python contributor.
+
+At the end of the ban, the developer is allowed to contribute again as a
+regular contributor. If the developer proved that their behaviour
+changed, another core developer can open a vote to propose to promote
```

Comment:
open a vote to propose the developer for promotion to core developer. The vote follows...